src/main/resources/static/main.css

```diff
@@ -271,4 +271,15 @@ footer {
     margin: 0.5em;
     margin-left: 1em;
     margin-right: 1em;
-}
\ No newline at end of file
+}
+
+#forceCheckIn{
```

Comment:
Passt hier nicht direkt in die PR, aber wenns einen guten Grund gibt musst auch nicht extra eine PR dafür aufmachen.

Reply:
Ja das hatte ich nur eingefügt, weil ich gesehen hab, dass es einen merge-conflict gibt ist aber wahrscheinlich falsch das dann an dieser Stelle einzufügen, habe es nochmal rausgenommen.